### ISSUE TITLE
Failed to build the app using MacOS

### ISSUE THREAD

**Issue by unsync** [vinceglb/ComposeAI#70]
Hi !
Is there any guide or working example to build the app ?

I've been trying to build it using a M1 Macbook with Fleet/IntelliJ/Android Studio, and completely failing ^^

So far i tried running the gradle task `build` for the shared module, here is the results :

### The first issue (using main branch)

```
> Task :shared:compileKotlinIosArm64 FAILED
e: Could not find "co.touchlab:stately-common" in [/Users/nico/Downloads/workspace/perso/others/ComposeAI/shared, /Users/nico/.konan/klib, /Users/nico/.konan/kotlin-native-prebuilt-macos-aarch64-1.9.22/klib/common, /Users/nico/.konan/kotlin-native-prebuilt-macos-aarch64-1.9.22/klib/platform/ios_arm64]

Could not find "co.touchlab:stately-common" in [/Users/nico/Downloads/workspace/perso/others/ComposeAI/shared, /Users/nico/.konan/klib, /Users/nico/.konan/kotlin-native-prebuilt-macos-aarch64-1.9.22/klib/common, /Users/nico/.konan/kotlin-native-prebuilt-macos-aarch64-1.9.22/klib/platform/ios_arm64]
```

I added the dependency in the shared module : 
```
// Stately
implementation(libs.stately.common)
```

### Second issue
```
> Task :shared:compileIosMainKotlinMetadata FAILED
ERROR: Exception while analyzing expression in (5,54) in /Users/nico/Downloads/workspace/perso/others/ComposeAI/shared/src/iosMain/kotlin/main.ios.kt

Attachments:
causeThrowable
java.lang.NullPointerException
```

Other error :

```
duplicate library name: runtime_commonMain
```

![Screenshot 2024-03-26 at 17 30 58](https://github.com/vinceglb/ComposeAI/assets/1211591/125844b5-879d-407a-8cb0-3251e4c4d507)

This seems (maybe) related to sqldelight : https://youtrack.jetbrains.com/issue/KT-60761

Long story short, could it be possible to have an example of the build steps and IDE used to build the app ?

Thanks a lot, can't wait to test this :)

**Comment by vinceglb** [comment]
Some time ago, I broke the iOS build to only focus on the Android side.

I fixed the iOS version in the #71 🎉  I tested on Android Studio and Fleet on macOS, and it works nice on Android and iOS 👌 

I merged into main. Tell me if everything is working for you now

**Comment by unsync** [comment]
@vinceglb thanks for this quick feedback !

Now i'm getting this when opening `iosApp` in XCode : 

```
Project ComposeAI/iosApp/iosApp.xcodeproj cannot be opened because it is missing its project.pbxproj file.
```

I've noticed there is two `.xcodeproj` files in the folder, is that expected ?

![Screenshot 2024-03-27 at 14 05 59](https://github.com/vinceglb/ComposeAI/assets/1211591/58653e7a-242d-4ad5-8cd0-0943fca02d56)

I'm kind of a newbie with multiplatform, tried it before but didn't get this far. I need to open the project in xcode first to setup the build right ?

**Comment by unsync** [comment]
@vinceglb i confirm the build works :)

I'll now be able to explore this app, very nice work 👏 

While i'm there, there seem to be an issue with the scroll on iOS, when i start a chat i cannot go back to the top (not sure how to describe this properly ^^), this is what is see:

![IMG_F22D919A31C5-1](https://github.com/vinceglb/ComposeAI/assets/1211591/7cdb7bea-6b37-45bb-a87c-39a06c548f55)

**Comment by vinceglb** [comment]
Cool! Yea, since I didn't focus on the iOS side, there are probably multiple obvious bugs like yours. 

If you find a way to fix it, feel free to open a PR if you want 😄

**Comment by unsync** [comment]
Will do 👍
